presto-main/src/main/java/com/facebook/presto/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -347,7 +347,7 @@ else if (partitioningHandle.equals(SCALED_WRITER_DISTRIBUTION)) {
                         bucketNodeMap,
                         splitBatchSize,
                         getConcurrentLifespansPerNode(session),
-                        nodeScheduler.createNodeSelector(null),
+                        nodeScheduler.createNodeSelector(connectorId),
```

Comment:
Is this causing any issues currently?

Reply:
No. @dain noted my new written code should use `connectorId`: https://github.com/prestodb/presto/pull/11693#discussion_r226462652 .  And this applied to existing  code as well.